tools/breaking-changes/generate-doc.ts

```diff
@@ -128,17 +160,29 @@ function getMembersDoc(apiElement: any): string {
           doc += `
 
 Previous version: 
-${MD_CODEBLOCK}${memberBreakingChange.previousStateDoc}${MD_CODEBLOCK}
+${MD_CODEBLOCK}${common.getMemberStateDoc(
+            memberBreakingChange.old
+          )}${MD_CODEBLOCK}
 
 Current version: 
-${MD_CODEBLOCK}${memberBreakingChange.currentStateDoc}${MD_CODEBLOCK}
+${MD_CODEBLOCK}${common.getMemberStateDoc(
+            memberBreakingChange.new
+          )}${MD_CODEBLOCK}
 `;
-
+          if (!!memberBreakingChange.migrationComment) {
+            doc += `\n${memberBreakingChange.migrationComment}\n`;
+          }
           break;
         }
         case 'DELETED': {
-          doc += `\n### ${memberBreakingChange.deletedMember.kind} ${memberBreakingChange.deletedMember.name} is removed.\n`;
-          doc += `\n${memberBreakingChange.migrationComment}\n`;
+          const memberMigrationComment = common.findDeletedMemberComment(
+            apiElement,
+            memberBreakingChange.changeElementName,
+            deletedMembersCommentData
+          );
+
+          doc += `\n### ${memberBreakingChange.old.kind} ${memberBreakingChange.old.name} is removed.\n`;
+          doc += `\n${memberMigrationComment}\n`;
```

Comment:
what do you think about prefixing these 'additional' comments as shown below. (Valid to all instances in this generated doc automation - reason is it gets appended below the generated 'automatic' comments, but then there's this random text that appears below, which is weird (in my opinion))
```suggestion
          doc += `\nAdditional comments: ${memberMigrationComment}\n`;
```